viewer/src/main/webapp/viewer-html/components/EditBulk.js

```diff
@@ -0,0 +1,396 @@
+            return viewer.components.EditBulk.superclass.featuresReceived.call(this, features);
+        },
+        handleFeature: function (feature) {
+            if (feature !== null) {
+
+                if (this.mode === "copy") {
+                    this.currentFID = null;
+                } else {
+                    this.currentFID = feature.__fid;
+                }
+
+                if (viewer.components.Component.parseBooleanValue(this.appLayer.details["editfeature.uploadDocument"])) {
+                    this.buildUploadedFilesRemovers(feature);
+                }
+                if (this.geometryEditable) {
+                    // var wkt = feature[this.appLayer.geometryAttribute];
+                    // var feat = Ext.create("viewer.viewercontroller.controller.Feature", {
+                    //     wktgeom: wkt,
+                    //     id: "T_0"
+                    // });
+                    // this.vectorLayer.addFeature(feat);
+                    this.showAndFocusForm();
+                } else {
+                    this.showAndFocusForm();
+                }
+
+                this.selectOrDeselectFeature(feature);
+            }
+            this.getContentContainer().unmask();
+        },
+        selectOrDeselectFeature: function (feature) {
+            if (this.selectedFeatures.isSelected(feature)) {
+                this.selectedFeatures.deselect(feature);
+            } else {
+                this.selectedFeatures.select(feature, this.lastClickCoords);
+            }
+
+            if (this.selectedFeatures.zeroSelected()) {
+                this.setFormVisible(false);
+            }
+
+            if (!this.selectedFeatures.zeroSelected()) {
+                this.setFormValues(feature);
+            }
+        },
+        setFormValues: function (feature_arg_will_not_be_used) {
+            var me = this;
+
+            this.formValuesAreBeingUpdated = true;
+
+            var values = this.selectedFeatures.getCombinedAttributeValues();
+            var attributeNames = this.getListOfAttributeNamesThatAreMappedToFormFields();
+            attributeNames.forEach(function(attributeName) {
+                var value = values[attributeName];
+
+                if (value === MULTIPLE_VALUES) {
+                    me.makeFormFieldMultiple(attributeName);
+                } else {
+                    me.makeFormFieldScalar(attributeName);
+                    var input = me.getFormFieldByAttributeName(attributeName);
+                    input.setValue(value);
+                }
+            });
+
+            this.formValuesAreBeingUpdated = false;
+        },
+        getFormFieldByAttributeName: function(attributeName) {
+            return this.inputContainer.getForm().findField(attributeName);
+        },
+        cancel: function () {
+            this.selectedFeatures.deselectAll();
+            return viewer.components.EditBulk.superclass.cancel.call(this);
```

Comment:
When doing some edits, saving and then creating a new feature, the previous selections are still there

Reply:
Not yet resolved

Reply:
Could you explain me how to reproduce this?

Reply:
Yes: Select 2 features, edit one attribute (I have two editable attributes) and save. Then, create a new feature. The previously edited attribute is empty (as expected), the other one has the values of previous selection.

Reply:
resolved by disabling the possibility of adding new features through the EditBulk component